web/src/main/java/com/bazaarvoice/emodb/web/scanner/writer/AmazonS3Provider.java

```diff
@@ -82,8 +83,10 @@ public AmazonS3 getS3ClientForBucket(String bucket) {
      * necessity to code in the s3 endpoint pattern, but this is extremely unlikely to change.
      */
     public AmazonS3 createS3ClientForRegion(String regionName) {
-        return new AmazonS3Client(_credentialsProvider)
-                .withEndpoint(String.format("s3.%s.amazonaws.com", regionName));
+        AmazonS3  s3;
+        s3 = AmazonS3ClientBuilder.standard().withCredentials(_credentialsProvider)
+                .withRegion(String.format("s3.%s.amazonaws.com", regionName)).build();
+        return s3;
```

Comment:
I think 3 lines are unnecessary.  We can have single line like this
```suggestion
        return AmazonS3ClientBuilder.standard().withCredentials(_credentialsProvider)
                .withRegion(String.format("s3.%s.amazonaws.com", regionName)).build();
```